bundle/config/mutator/resourcemutator/apply_bundle_permissions.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/databricks/cli/libs/dyn/convert"
 )
 
-var unsupportedResources = []string{"clusters", "volumes", "schemas", "quality_monitors", "registered_models", "database_catalogs", "synced_database_tables"}
+var unsupportedResources = []string{"volumes", "schemas", "quality_monitors", "registered_models", "database_catalogs", "synced_database_tables"}
```

Comment:
This should come with a comment about why they're unsupported (i.e., because they're in unity catalog and use GRANTs rather than permissions)

Reply:
Makes sense. This is only used in tests as completeness check, so I'll move it there as well.